### PR TITLE
Packaging: allow any version

### PR DESCRIPTION
**Pull request by DmitryZabolotsky**
Close #36

Also updated changelog

### CODE REVIEW

**Review by acid-ant — DISMISSED** [review]

**Comment by realloc** [comment]
This would cause each package to be marked as -dirty. AFAIR, previously we agreed to carefully set version corresponding to a particular release to avoid false dirty builds.

File: debian/changelog [hunk 1]
```diff
@@ -1,18 +1,5 @@
-frostfs-node (0.34.0~97-dirty~g42554a92-jammy) jammy; urgency=medium
```
Comment (fyrchik) [inline debian/changelog:1]: When do we need to change this number? Every release of a `frostfs-node`? On every non-compatible change?

Comment (fyrchik) [inline debian/changelog:1]: When do we need to change this number? Every release of a frostfs-node? On every non-compatible change?

Comment (realloc) [reply debian/changelog:1]: Each package build must have package version to be the same as one reported by `--version` from cli.

**Review by alexvanin — APPROVED** [review]
This change is required to use latest FrostFS version in CI pipeline. So it would be nice to merge it soon.

**Review by realloc — APPROVED** [review]
Did this fix the dirty suffix in the resulting binary inside package?

**Review by acid-ant — APPROVED** [review]

**Pull request merged**